app/0.3.x/getting-started/introduction.md

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
Why is a whole new file appearing? This is curious to me.

Reply:
Nothing changed, not sure why git is being strange here.